From 218df5b7fc3c713ffcdf7f3709aab0171c75a473 Mon Sep 17 00:00:00 2001
From: Luis Medinas <lmedinas@gnome.org>
Date: Sat, 5 Feb 2011 16:55:10 +0000
Subject: [PATCH] Fix Bug 641407 - libbrasero-media3.pc doesn't work

Libbrasero-media3 and libbrasero-burn3 had a typo in gtk+-3.0 call.
---
 libbrasero-burn3.pc.in  |    2 +-
 libbrasero-media3.pc.in |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/libbrasero-burn3.pc.in b/libbrasero-burn3.pc.in
index 456ba00..2937efe 100644
--- a/libbrasero-burn3.pc.in
+++ b/libbrasero-burn3.pc.in
@@ -7,6 +7,6 @@ includedir=@includedir@
 Name: libbrasero-burn
 Description: Brasero Optical Media Burning library
 Version: @VERSION@
-Requires: glib-2.0 gthread-2.0 gtk+-3
+Requires: glib-2.0 gthread-2.0 gtk+-3.0
 Libs: -L${libdir} -lbrasero-burn3
 Cflags: -I${includedir}/brasero3
diff --git a/libbrasero-media3.pc.in b/libbrasero-media3.pc.in
index aacddc7..5f303f8 100644
--- a/libbrasero-media3.pc.in
+++ b/libbrasero-media3.pc.in
@@ -7,6 +7,6 @@ includedir=@includedir@
 Name: libbrasero-media
 Description: Brasero Optical Media library
 Version: @VERSION@
-Requires: glib-2.0 gthread-2.0 gtk+-3
+Requires: glib-2.0 gthread-2.0 gtk+-3.0
 Libs: -L${libdir} -lbrasero-media3
 Cflags: -I${includedir}/brasero3
-- 
1.7.4

